litmus-portal/frontend/src/views/Analytics/DataSources/Table/TableData.tsx

```diff
@@ -105,14 +106,15 @@ const TableData: React.FC<TableDataProps> = ({
       </StyledTableCell>
       <StyledTableCell className={classes.columnDivider}>
         <Typography
-          className={`${classes.tableObjects} ${classes.dataSourceNameColData}`}
+          className={`${classes.tableObjects} ${classes.dataSourceNameCol}`}
+          style={{ fontWeight: 500 }}
```

Comment:
remove inline styles

Reply:
here we needed to override a single property so it has been used this way